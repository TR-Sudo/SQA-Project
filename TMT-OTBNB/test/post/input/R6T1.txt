login
user01
post
Oshawa
110
five
5
logout
quit