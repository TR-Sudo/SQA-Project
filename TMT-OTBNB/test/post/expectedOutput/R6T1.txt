Enter transaction: Enter username: You have been logged in
Enter transaction: Enter City Name: Enter rental price (0-999.99): Enter number of rooms (0-9): Error: Invalid number
Enter number of rooms (0-9): Unit posted by user: user01
Enter transaction: You have been logged out
Enter transaction: 